plugins/codemirror/plugin.js

```diff
@@ -4,6 +4,47 @@
 	var stylesLoaded = false;
 
 	CKEDITOR.plugins.add('codemirror', {
+		_addTabKeyMessage: function (cm, editor, tabsKeyIsEnabled) {
+			var div = document.createElement('div');
+
+			var message = tabsKeyIsEnabled
+				? editor.lang.codemirror.disableTabKeyUsing
+				: editor.lang.codemirror.enableTabKeyUsing;
+
+			var html =
+				'<div class="keyboard-message popover px-2 py-1" style="left:auto; right: 4px; top:4px">';
```

Comment:
This introduces some clay-css dependencies, whick may not be available in all cases when using the editor in non-admin contexts. The bigger issue seems with the popover class as it defines the zindex and positioning. 

Probably with some inlining this can be worked around